project.clj

```diff
@@ -57,7 +57,7 @@
                  [com.jolbox/bonecp "0.7.1.RELEASE" :exclusions [org.slf4j/slf4j-api]]
                  [org.clojure/java.jdbc "0.3.7"]
                  [org.hsqldb/hsqldb "2.2.8"]
-                 [org.postgresql/postgresql "9.2-1003-jdbc4"]
+                 [org.postgresql/postgresql "9.4.1208"]
```

Comment:
General upkeep but also fixes https://tickets.puppetlabs.com/browse/PDB-1937 for users sooner